src/user/models/user/dto/linked-user.dto.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+import { IsBoolean, IsEnum, IsNotEmpty, IsString, Matches } from 'class-validator';
+import { Blockchain } from 'src/ain/services/crypto.service';
+import { GetConfig } from 'src/config/config';
+
+export class LinkedUserInDto {
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsString()
+  @Matches(GetConfig().addressFormat)
+  address: string;
+
+  @ApiProperty()
+  @IsNotEmpty()
+  @IsEnum(Blockchain)
+  blockchain: Blockchain;
+}
+
+export class LinkedUserOutDto extends LinkedUserInDto {
+  @ApiPropertyOptional()
+  @IsBoolean()
```

Comment:
`@ApiProperty()` (non nullable) and remove `@IsBoolean()` (no validation on out DTO)

Reply:
oh true - just copy pasted